From 414f1ddd402d481ff740901376a3eee9736e4a65 Mon Sep 17 00:00:00 2001
From: Dave Young <dyoung@redhat.com>
Date: Wed, 28 May 2014 07:40:05 -0400
Subject: [redhat] Kconfig: explicitly set CONFIG_EFI_RUNTIME_MAP=y on x86_64

Message-id: <20140528074005.GA16713@dhcp-16-198.nay.redhat.com>
Patchwork-id: 80937
O-Subject: [RHEL7.1 PATCH 36/35] Kconfig: x86_64 explicitly set CONFIG_EFI_RUNTIME_MAP=y
Bugzilla: 1080109
RH-Acked-by: Chao WANG <chaowang@redhat.com>
RH-Acked-by: Vivek Goyal <vgoyal@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

Resolves: bz1080109

This option is for UEFI support in kexec/kdump kernel.

Suggested by Jarod and Don, quote comments from Jarod below:
"""
I'd advocate for setting it explicitly as well, for multiple reasons. If
its only turned on because some other setting turns it on, it could get
unexpectedly turned off if the other setting changes or goes away, and
come RHEL8, it would useful to have something in the git commit logs to go
on explaining why its on, so we can make a better educated decision for
RHEL8, should questions arise about that option.
"""
Brew build:
https://brewweb.devel.redhat.com/taskinfo?taskID=7508649

Tested with `make rh-configs-prep`
Tested kexec/kdump on qemu.

Signed-off-by: Dave Young <dyoung@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_EFI_RUNTIME_MAP b/redhat/configs/generic/x86_64/CONFIG_EFI_RUNTIME_MAP
new file mode 100644
index 0000000..3a4462f
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_EFI_RUNTIME_MAP
@@ -0,0 +1 @@
+CONFIG_EFI_RUNTIME_MAP=y
-- 
1.7.1